tests/wandb_tensorflow_test.py

```diff
@@ -153,3 +154,53 @@ def test_compat_tensorboard(live_mock_server, test_settings):
     print(first_stream_hist)
     assert json.loads(first_stream_hist["content"][-1])["_step"] == 9
     assert json.loads(first_stream_hist["content"][-1])["global_step"] == 9
+    assert (
+        "\x1b[34m\x1b[1mwandb\x1b[0m: \x1b[33mWARNING\x1b[0m Step cannot be set when using"
+        " syncing with tensorboard. Please log your step values as a metric such as 'global step'"
```

Comment:
This should be "global_step" so it would match whatever tensorboard is logging the step as.